app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::CommentsController < ApiController
+  def index
+    @comments = Post.find(params[:post_id]).comments
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.post_id = params[:post_id]
+    @comment.user = current_user
+
+    if @comment.save
+      render json: { success: ['Comment added successfully'] }
+    else
+      render json: { success: ['Comment not created'] }
+    end
+  end
```

Comment:
- [ ] `Create method` for comment doesn't work as expected. Check for render json message .

![user-8](https://user-images.githubusercontent.com/55635977/139907713-061b2e58-1705-40c4-8286-38f5a8202399.png)
![user-9](https://user-images.githubusercontent.com/55635977/139907728-3dd5bc36-2047-47e7-a620-c73341fd4ade.png)

Reply:
Hello @Emmyn5600, this is because the ID used doesn't exist. There is no post with that ID

Reply:
Yeah I see, the second one can’t work because we didn’t create a post yet

Reply:
So if you finished the create method, it will be easy to get the post that has been created